src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -21,7 +21,7 @@
 def create_target_image(cmd, location, transient_resource_group_name, source_type, source_object_name,
                         source_os_disk_snapshot_name, source_os_disk_snapshot_url, source_os_type,
                         target_resource_group_name, azure_pool_frequency, tags, target_name, target_subscription,
-                        export_as_snapshot, timeout, hyper_v_generation='V1'):
+                        export_as_snapshot, timeout, hyper_v_generation=None):

```

Comment:
May I ask why do we need to modify the default value? Please note that modifying the default value of parameters may cause breaking change to users

Reply:
> May I ask why do we need to modify the default value? Please note that modifying the default value of parameters may cause breaking change to users

okay, I will restore the default value to `V1`